webvtt/api/VTTCue/region.html

```diff
@@ -10,7 +10,7 @@
     document.body.appendChild(video);
 
     var cue = new VTTCue(0, 1, 'text');
-    assert_equals(cue.region, null);
```

Comment:
The original `assert_equals(cue.region, null)` should still be checked, since `undefined !== null` and the constructor should set `region` to `null`